scripts/jenkins/jobs-ibs/openstack-mkcloud.yaml

```diff
@@ -0,0 +1,508 @@
+
+    logrotate:
+        numToKeep: 1000
+        daysToKeep: 300
+
+    properties:
+      - authorization:
+          cloud:
+               - job-build
+               - job-cancel
+               - job-configure
+               - job-delete
+               - job-discover
+               - job-read
+               - job-workspace
+               - run-delete
+               - run-update
+               - scm-tag
+          anonymous:
+               - job-read
+
+    parameters:
+     - string:
+        name: cloudsource
+        default: develcloud7
+        description: |
+             Can be something like develcloud6 (uses Devel:Cloud:*, see TESTHEAD parameter for Staging),  susecloud6 (uses SUSE:*), GM3 (uses the GM ISO)
+
+     - string:
+        name: mkcloudtarget
+        default: all
+        description: Set mkcloudtarget to define the mkcloud command line parameters.
+
+     - choice:
+         name: libvirt_type
+         choices:
+           - kvm
+           - xen
+           - hyperv   
+
+     - choice:
+         name: networkingplugin
+         choices:
+            - openvswitch
+            - linuxbridge
+
+     - choice:
+         name: networkingmode
+         choices:
+             - gre
+             - vlan
+             - vxlan
+
+     - choice:
+         name: storage_method
+         description: |
+              Select storage method: default (let mkcloud decide), none (enforce none)
+         choices:
+            - default
+            - none
+            - swift
+            - ceph
+ 
+     - string:
+         name: github_pr
+         description: |
+              String is a ':' separated list of these vaules: $ORG/$repo:$PR_ID:$SHA1:$BRANCH:$context Note: 
+              SHA1 must be latest commit in PR, $context is optional  (only use if multiple gating runs needed per PR)
+    
+     - string:
+         name: UPDATEREPOS
+         default:
+         description: |
+              Update repositories (one URL per line)
+
+     - bool:
+         name: UPDATEBEFOREINSTALL
+         default: false
+         description: 'add update repos before crowbar install'
+     
+     - string:
+         name: TESTHEAD
+         default : '1'
+         description: |
+              if non-empty, test latest version from Devel:Cloud (:1.0)
+   
+     - string:
+         name: nodenumber
+         default: '2'
+         description: number of nodes excl. admin-node minimum 2=controller+compute
+
+     - string:
+         name: nodenumbercompute
+         default:
+         description: Number of compute nodes
+
+     - string: 
+         name: want_tempest
+         default: 
+         description: can be used for force off(0) or on(1)
+
+     - string: 
+         name: tempestoptions
+         default: -t -s
+         description: |
+           Additional options to pass to the "run_tempest.sh -N " step.
+           Use just \"-t\" for a full tempest run.
+           Default: -t -s.
```

Comment:
nit: no need to write the defaults in the description I think. It's written 4 lines above